e2e-test/src/test/java/io/cdap/plugin/sendEmail/stepsdesign/SendEmail.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.sendEmail.stepsdesign;
+import io.cdap.e2e.pages.actions.CdfPluginPropertiesActions;
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.plugin.sendEmail.actions.SendEmailActions;
+import io.cdap.plugin.utils.GmailClient;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import javax.mail.MessagingException;
+
+/**
+ * Send-email alert plugin related step design.
+ */
+
+public class SendEmail  {
+
+  @And("Select send email alert plugin")
+  public void selectSendEmailAlert() {
+    SendEmailActions.clickOnSendEmail();
+  }
+
+  @And("And Enter send email plugin property email recipient with value: {string}")
+  public void enterInputPluginPropertyWithValue(String value) {
```

Comment:
enterInputPluginPropertyWithValue -> enterSendEmailPluginPropertyEmailRecipientWithValue

Best practice is whenever you change step description-create step definition from scenario file. It will automatically creates proper method names according to description